docs/source/frameworks/fastai.rst

```diff
@@ -0,0 +1,192 @@
+   from fastai.data.transforms import get_text_files
+   from fastai.data.transforms import GrandparentSplitter
+
+   # Download IMDB dataset
+   path = untar_data(URLs.IMDB)
+
+   # Create IMDB DataBlock
+   imdb = DataBlock(
+       blocks=(TextBlock.from_folder(path), CategoryBlock),
+       get_items=get_text_files,
+       get_y=parent_label,
+       splitter=GrandparentSplitter(valid_name="test"),
+   )
+   dls = imdb.dataloaders(path)
+
+   # define a Learner object
+   learner = text_classifier_learner(
+        dls, AWD_LSTM, drop_mult=0.5, metrics=accuracy
+    )
+
+   # quickly fine tune the model
+   learner.fine_tune(4, 1e-2)
+
+   # output:
+   # epoch     train_loss  valid_loss  accuracy  time
+   # 0         0.453252    0.395130    0.822080  36:45
+
+   learner.predict("I really liked that movie!")
+
+   # output:
+   # ('pos', TensorText(1), TensorText([0.1216, 0.8784]))
+
+
+After training, use :obj:`~bentoml.fastai.save_model` to save the `Learner <https://docs.fast.ai/learner.html#Learner>`_ instance to BentoML model store.
+
+.. code-block:: python
+
+   bentoml.fastai.save_model("fastai_sentiment", learner)
+
+
+
+To verify that the saved learner can be loaded properly:
+
+.. code-block:: python
+
+   learner = bentoml.fastai.load_model("fastai_sentiment:latest")
+
+   learner.predict("I really liked that movie!")
+
+
+Building a Service using fastai
+--------------------------------
+
+.. seealso::
+
+   :ref:`Building a Service <concepts/service:Service and APIs>`: more information on creating a prediction service with BentoML.
+
+.. code-block:: python
+
+   import bentoml
+
+   import numpy as np
+
+   from bentoml.io import Text
+   from bentoml.io import NumpyNdarray
+
+   runner = bentoml.fastai.get("fastai_sentiment:latest").to_runner()
+
+   svc = bentoml.Service("fast_sentiment", runners=[runner])
+
+
+   @svc.api(input=Text(), output=NumpyNdarray())
+   def classify_text(text: str) -> np.ndarray:
+      # returns sentiment score of a given text
+      res = runner.predict.run(text)
+      return np.asarray(res[-1])
+
+
+When constructing a :ref:`bentofile.yaml <concepts/bento:Bento Build Options>`,
+there are two ways to include fastai as a dependency, via ``python`` or
+``conda``:
+
+.. tab-set::
+
+   .. tab-item:: python
+
+      .. code-block:: yaml
+
+         python:
+         - fastai
+
+   .. tab-item:: conda
+
+      .. code-block:: yaml
+
+         conda:
+           channels:
+           - fastchan
+           dependencies:
+           - fastai
+
+
+Using Runners
+-------------
+
+.. seealso::
+
+   See :ref:`concepts/runner:Using Runners` doc for a general introduction to the Runner concept and its usage.
+
+
+``runner.predict.run`` is generally a drop-in replacement for ``learner.predict`` regardless of the learner type.
+
+A fastai :obj:`~bentoml.Runner` is a wrapper around a fastai :~obj:`Learner`
+object. This means that a fastai runner will receive the same inputs type as
+the given learner.
```

Comment:
Calling runner a wrapper around the fastai learner may be an incorrect description. Maybe we can expand the previous paragraph and remove this one.
```suggestion
```